config/config.go

```diff
@@ -74,7 +82,7 @@ var (
 func DefaultConfig() *Config {
 	return &Config{
 		AssetRegistryScheme: "https",
-		AssetRegistryDomain: "testnet-assets.rddl.io",
+		AssetRegistryDomain: "assets.rddl.io",
```

Comment:
Please revert this diff. Testnet is the default: did you forget to rebase/merge main?